app/scripts/modules/core/src/managed/ManagedResourceStatusIndicator.tsx

```diff
@@ -177,30 +185,125 @@ const logClick = (label: string, resourceId: string, status: ManagedResourceStat
     label: `${resourceId},${status}`,
   });
 
-const LearnMoreLink = ({ id, status }: { id: string; status: ManagedResourceStatus }) => (
-  <p className="sp-margin-m-top sp-margin-xs-bottom">
-    <a
-      target="_blank"
-      onClick={() => logClick('Status docs link', id, status)}
-      href={`https://www.spinnaker.io/reference/managed-delivery/resource-status/#${status.toLowerCase()}`}
-    >
-      Learn more about this
-    </a>
-  </p>
+const toggleResourcePause = (
+  resourceSummary: IManagedResourceSummary,
+  application: Application,
+  hidePopover: () => void,
+) => {
+  hidePopover();
+  const { id, isPaused } = resourceSummary;
+  const toggle = () =>
+    isPaused ? ManagedWriter.resumeResourceManagement(id) : ManagedWriter.pauseResourceManagement(id);
+
+  const submitMethod = () => toggle().then(() => application.managedResources.refresh(true));
+
+  return ReactInjector.confirmationModalService.confirm({
+    header: `Really ${isPaused ? 'resume' : 'pause'} resource management?`,
+    body: getToggleBodyText(resourceSummary),
+    account: resourceSummary.locations.account,
+    buttonText: `${isPaused ? 'Resume' : 'Pause'} management`,
+    submitMethod,
+  });
+};
+
+const getToggleBodyText = (resourceSummary: IManagedResourceSummary) => {
+  const { isPaused, locations, status } = resourceSummary;
+  const regions = locations.regions.map(r => r.name).sort();
+  let body = '';
+  if (!isPaused) {
+    body += `
+        <p>
+          While a resource is paused, Spinnaker will not attempt to reconcile any differences between the declared
+          configuration and the current state of the resource.
+        </p>`;
+    body += viewConfigurationByStatus[status]?.pauseWarning ?? '';
+  } else {
+    body += `
+        <p>
+          Spinnaker will resume reconciling any differences between the declared configuration and the current state of
+          the resource.
+        </p>
+      `;
+  }
+  if (regions.length > 1) {
+    body += `
+        <p>
+          <div class="horizontal top sp-padding-m alert alert-warning">
+            <i class="fa fa-exclamation-triangle sp-margin-m-right sp-margin-xs-top"></i>
+            <span>${isPaused ? 'Resuming' : 'Pausing'} management of this resource will ${isPaused ? 'resume' : 'pause'}
+             management of this resource in the following managed regions: <b>${regions.join(', ')}</b>.
```

Comment:
"Resuming management of this resource will resume management of this resource in.." is pretty repetitive — what about "Resuming management of this resource will affect the following regions:"?